pkg/api/grpcServer.go

```diff
@@ -61,9 +61,119 @@ type (
 )
 
 var (
```

Comment:
Is there a compelling reason to hard code this over ingesting it as a yaml file? Then we can just use the global_dictonary.yaml directly. Eventually we'll need to manage rolling out changes to this list (and probably handle having multiple versions to serve with at the same time), so we'll want it to not be compiled in anyway.